.github/workflows/go.yml

```diff
@@ -10,6 +10,9 @@ on:
     - main
     - release-*
     - feature/*
+  schedule:
```

Comment:
I think you need a separate workflow to do this kind of daily check instead of adding schedule here. @antoninbas any suggestion?

Reply:
done.